TAG-S&P-questionnaire.md

```diff
@@ -0,0 +1,106 @@
+# Self-Review Questionnaire: Security and Privacy
+
+Questions from w3.org/TR/security-privacy-questionnaire
+
+## 2. Questions to Consider
+
+### 2.1. What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?
+
+If unpartitioned third-party cookies are still in use, this feature does not expose any additional information that is not already available if sites were already using an unpartitioned cross-site cookie.
+If unpartitioned third-party cookies are blocked, this provides sites in third-party contexts the ability to use HTTP cookies that are scoped to a single top-level site.
+
+### 2.2. Is this specification exposing the minimum amount of information necessary to power the feature?
+
+Yes. This proposal gives the ability for third-party contexts the ability to use a familiar HTTP state primitive for use cases restricted to a users' activity in only the current top-level site.
```

Comment:
Would it make sense to mention that HTTP State Tokens expose a bit less information, but that they seem to require too much effort from site operators to adopt?

Reply:
How do they reveal less information? Is it because they are client generated and their value is not set by the site?

Reply:
Yeah, so there's less space total and no worry about the site putting anything sensitive there.

Reply:
Got it. Ok, added something about HST's here.